README.md

```diff
@@ -62,6 +62,14 @@ type ProviderInterface interface {
 ```
 
 Prior to the build, run `pina-golada generate` to create a generated Go source file, which contains your assets. To clean up any generated file, use `pina-golada cleanup`. If you fancy more details, enable the verbose mode using `--verbose`. This works for `generate` as well as `cleanup`.
+In addition to the verbose parameter, you can specify the annotation `parser` you want to use by providing the parser tag value.
+Here are the currently supported `parser` values:
```

Comment:
Markdown linter prefer an empty line between the text and the list. Do not ask me why. I usually try to please the linter.